### ISSUE TITLE
best way to add ITU information

### ISSUE THREAD

**Issue by armijnhemel** [kaitai-io/kaitai_struct_formats#464]
I am looking at some specifications that have been published by ITU. I would like to add them to the meta field but I could not find any examples. What is the best way to add these?

**Comment by KOLANICH** [comment]
``yaml
meta:
  xref:
    itu: ...
```

;)

**Comment by armijnhemel** [comment]
> ```yaml
> meta:
>   xref:
>     itu: ...
> ```
> 
> ;)

Ok, let me rephrase: what would the preferred format be? Just the number of the ITU specification. or something else?

**Comment by KOLANICH** [comment]
With the letter.
